presto-native-execution/README.md

```diff
@@ -109,4 +109,4 @@ export PATH=$HOME/bin:$HOME/Library/Python/3.7/bin:$PATH
 
 ### Setting up on Linux (CentOs 8 or later)
 
-See `scripts/setup-linux.sh`
+See `scripts/setup-circleci.sh`
```

Comment:
Is this the right fix? Shouldn't the script name be setup-ubuntu.sh ? CC: @spershin

Reply:
From this commit history, the setup-linux.sh was renamed to setup-circleci.sh.
https://github.com/facebookexternal/presto_cpp/commits/main/scripts/setup-circleci.sh

Reply:
CC: @mshang816 @spershin @majetideepak Should we rename setup-circleci.sh to setup-linux.sh to avoid confusion? It is strange to run setup-circleci.sh script in local environments.

Reply:
I think `setup-circleci.sh` is fine. `linux` is a general term used for various Linux distros (eg. `yum` vs `apt`). It can confuse users since we do not support all distros.

Reply:
Should we use a distro name then? setup-centos.sh?

Reply:
yeah `setup-centos.sh` is perfect!